src/runtime/nonce.ts

```diff
@@ -0,0 +1,9 @@
+import { plt } from '@platform';
+
+/**
+ * Assigns the given value to the nonce property on the runtime platform object.
+ * During runtime, this value is used to set the nonce attribute on all dynamically created script and style tags.
+ * @param nonce The value to be assigned to the platform nonce property.
+ * @returns void
+ */
+export const setNonce = (nonce: string) => (plt.$nonce$ = nonce);
```

Comment:
This is probably going to be covered by the docs, but is the assumption that users will have to change the nonce server-side for every request that's served?

Reply:
Sorta. I'm not sure the best way to phrase it, but the idea is that a nonce should be unique "per page view". So, for SPA like an Angular app, you can just generate a nonce at initial bootstrap and use that for the page's lifetime